Makefile

```diff
@@ -25,18 +25,16 @@ gce-pd-driver:
 	go build -o bin/gce-pd-csi-driver-test ./test/e2e/
 
 build-container: gce-pd-driver
-	cp bin/gce-pd-csi-driver deploy/docker
-	docker build -t $(STAGINGIMAGE):$(STAGINGVERSION) deploy/docker
+	docker build -t $(STAGINGIMAGE):$(STAGINGVERSION) .
```

Comment:
do you also want to remove references to your dev project and make that an env variable?

Reply:
done